File: Tugas coding modul 4/coding py/PRAK405-2310817120005-Noviana_Nur_Aisyah.py
x, y = map(int, input("Input \n").split())
sum = 0
sum2 = 0

print("\nOutput")
for i in range(1, x + 1):
    kali = i * y
    sum += kali
    if i == 1:
        print(f"({i} * {y}) = {kali}")
    else:
        for j in range(i, 0, -1):
            print(f"({j} * {y})", end="")
            if j > 1:
                print(" + ", end="")
        print(f" = {sum}")
    sum2 += sum

print(sum2)

File: Tugas coding modul 4/coding c/PRAK405-2310817120005-Noviana_Nur_Aisyah.c
#include <stdio.h>

void main(){
    int x, y, i, j, kali, sum, sum2;
    sum = 0, sum2 = 0;

    printf("Input \n");
    scanf("%d %d", &x, &y);

    printf("\nOutput \n");
    for (i = 1; i <= x; i++) {
        kali = i * y;
        sum += kali;
        if (i == 1) {
            printf("(%d * %d) = %d \n", i, y, kali);
        } else {
            for (j = i; j >= 1; j--) {
                printf("(%d * %d)", j, y);
                if (j > 1) {
                    printf(" + ");
                }
            }
            printf(" = %d \n", sum);
        }
        sum2 += sum;
    }
    printf("%d", sum2);
}